modules/ROOT/pages/installing-multiple-instances.adoc

```diff
@@ -0,0 +1,228 @@
+= Installing Multiple Instances of Runtime Fabric on a Single Cluster
+
+Installing multiple instances of Anypoint Runtime Fabric enables you to share the same cluster among multiple Runtime Fabrics, which helps you to use resources efficiently.
+
+[NOTE]
+To use this feature, you must upgrade to the minimum version of the agent that supports multiple instances.
+
+There will be 1:1 mapping for the agent namespace to the app namespace. Sharing app namespaces is not supported among multiple agent installations.
+
+== Install Multiple Instances of Runtime Fabric
+
+To install multiple instances of Anypoint Runtime Fabric on a single BYOK (Bring Your Own Kubernetes) cluster, follow these steps:
+
+. Create a Runtime Fabric instance using Runtime Manager.
+. Create a custom namespace to install the Runtime Fabric instance:
++ 
+----
+kubectl create ns <rtf_namespace>
```

Comment:
```suggestion
kubectl create ns [rtf_namespace]
Replace rtf_namespace with the name of namespace where Runtime Fabric will be installed.
```

Reply:
cc @SparshAgarwal, we like how google cloud documents it for e.g. at https://cloud.google.com/kubernetes-engine/docs/concepts/ingress

Reply:
let's standardize on using `[rtf_namespace]` as the placeholder